tests/actions.go

```diff
@@ -39,7 +39,7 @@ import (
 	pingcapErrors "github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
-	"github.com/pingcap/tidb-operator/pkg/client/clientset/versioned"
+	pcCli "github.com/pingcap/tidb-operator/pkg/client/clientset/versioned"
```

Comment:
Will it be greater if we spilt the name change from this PR? These changes seem not too related to the "upgrade tc version"

Reply:
yes, this could be in a separate. Let's see if anyone else thinks it deserves another PR, and make a change then?

Reply:
It's better to cherry-pick, and easier to support version management methods in this period. Some users can just pick-up this PR to their branch on-demand.